### PR TITLE
chore: Update contributing.adoc

### PR DESCRIPTION
**Pull request by themr0c**
* Refresh the accuracy of the contributing document.
* Fix vale errors.
* Add "Validate language changes".

### CODE REVIEW

**Review by yhontyk — APPROVED** [review]

File: CONTRIBUTING.adoc [hunk 1]
```diff
@@ -199,67 +213,54 @@ pass:[VARIABLE_NAME__WITH__UNDERSCORES]
...
-
-* An installation of https://github.com/linkchecker/linkchecker[linkchecker].
-
-
 .Procedure
 
-. Open a shell into the project directory.
-
-. Install Antora and Gulp using yarn:
+. Open a terminal into the project directory, and build the documentation using and instance of the `che-docs` container:
```
Comment (tstastna) [inline CONTRIBUTING.adoc:225]:
```suggestion
. Open a terminal into the project directory, and build the documentation using an instance of the `che-docs` container:
```

File: CONTRIBUTING.adoc [hunk 2]
```diff
@@ -199,67 +213,54 @@ pass:[VARIABLE_NAME__WITH__UNDERSCORES]
...
-+
-----
-$ gulp
+$ ./tools/runnerpreview.sh
 ---- 
 
 . Navigate to `+localhost:4000+` in your browser.
 
-. To create a new topic using, run following command:
+. Open a second terminal, and attach to the running `che-docs` container. Run all following steps from this context.
```
Comment (tstastna) [inline CONTRIBUTING.adoc:233]:
```suggestion
. Open a second terminal, and attach it to the running `che-docs` container. Run all the following commands from this context.
```

File: CONTRIBUTING.adoc [hunk 3]
```diff
@@ -199,67 +213,54 @@ pass:[VARIABLE_NAME__WITH__UNDERSCORES]
...
-. To validate the compliance of an Asciidoc file with Modular Documentation standards, execute following command from a `+bash+` terminal, from the root directory of the project:
+. To validate language on changed files, run:
 +
 ----
-$ bash ./tools/test-adoc.sh <PATH_TO_THE_FILE>
+$ ./tools/validate_language_changes.sh
 ----
 
-. To validate compliance of a file with the style guide, execute following command in a terminal, from the root directory of the project:
+. To create a new topic using templates, run following command:
```
Comment (tstastna) [inline CONTRIBUTING.adoc:251]:
```suggestion
. To create a new topic using the modular templates, run:
```

File: CONTRIBUTING.adoc [hunk 4]
```diff
@@ -4,45 +4,58 @@
...
+For Antora, a link:https://docs.antora.org/antora/2.3/module-directories/#module[_module_] is a collection of content in a component version. This project is referring to Modular Documentation modules as _topics_. To understand the nature of topics, see the link:https://redhat-documentation.github.io/modular-docs/#modular-docs-terms-definitions[Appendix A: Modular Documentation Terms and Definitions].
+====
+
+* Use the link:https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow[Git Forking Workflow].
 
-* link:https://www.atlassian.com/git/tutorials/comparing-workflows/forking-workflow[Git Forking Workflow].
+* link:https://www.eclipse.org/che/docs/che-7/hosted-che/[Eclipse Che hosted by Red Hat] is the recommended IDE to edit and builds the content.
 
-* link:https://www.eclipse.org/che/docs/che-7/hosted-che/[Eclipse Che hosted by Red Hat] edit and builds the content.
+* The relevant tools for all edition, validation and publication workflows are built in the link:https://quay.io/repository/eclipse/che-docs[`che-docs` container].
```
Comment (tstastna) [inline CONTRIBUTING.adoc]:
```suggestion
* The relevant tools for editing, validation, and publication workflows are built in the link:https://quay.io/repository/eclipse/che-docs[`che-docs` container].
```

File: CONTRIBUTING.adoc [hunk 5]
```diff
@@ -108,25 +121,26 @@ To build the Eclipse Che documentation from a Che workspace, follow the steps be
 
 .Procedure
 
-. Open _My Workspace_.
+. To build the documentation and start the preview server: open _My Workspace / User Runtimes / che-docs / Start preview server_.
+
+. To navigate the preview, open _Preview_.
```
Comment (tstastna) [inline CONTRIBUTING.adoc]:
```suggestion
. To navigate to the preview, open _Preview_.
```

File: CONTRIBUTING.adoc [hunk 6]
```diff
@@ -108,25 +121,26 @@ To build the Eclipse Che documentation from a Che workspace, follow the steps be
...
 
-. To process generated documentation: Open _User Runtimes / bash-curl / Generate reference tables_.
+. To validate links: open _User Runtimes / che-docs / Validate links_.
```
Comment (tstastna) [inline CONTRIBUTING.adoc]:
```suggestion
. To validate the links: open _User Runtimes / che-docs / Validate links_.
```

File: CONTRIBUTING.adoc [hunk 7]
```diff
@@ -108,25 +121,26 @@ To build the Eclipse Che documentation from a Che workspace, follow the steps be
...
 
-. To build the documentation and start the preview server: open _User Runtimes / che-docs-dev / Start preview server_.
+. To validate language changes: open _My Workspace / User Runtimes / che-docs / Validate language changes_.
```
Comment (tstastna) [inline CONTRIBUTING.adoc]:
```suggestion
. To validate the language changes: open _My Workspace / User Runtimes / che-docs / Validate language changes_.
```

File: CONTRIBUTING.adoc [hunk 8]
```diff
@@ -108,25 +121,26 @@ To build the Eclipse Che documentation from a Che workspace, follow the steps be
...
 
-. To validate the language of the files currently open, look at the _Problems_ panel in the _Bottom Panel_. To toggle the view of this panel use the _View > Problems_ menu entry.
+. To validate the language of the files currently open, look at the _Problems_ tab in the _Bottom Panel_. To toggle the view of this tab use the _View > Problems_ menu entry.
```
Comment (tstastna) [inline CONTRIBUTING.adoc]:
```suggestion
. To validate the language of the currently open files, look at the _Problems_ tab in the _Bottom Panel_. To toggle the view of this tab use the _View > Problems_ menu entry.
```

**Review by tstastna — APPROVED** [review]

**Review by sleshchenko — APPROVED** [review]

**Comment by sleshchenko** [comment]
note that https://github.com/eclipse/che-docs/pull/2005/checks?check_run_id=2674645650 fails

**Comment by themr0c** [comment]
Fixed the last vale errors.

**Pull request merged**